templates/extra/faq.html

```diff
@@ -5,16 +5,16 @@
     <div class="break">
         <hr>
     </div>
-    <div class="extra-padding">
+    <div class="faq-padding">
         <ul class="vertical menu" data-accordion-menu>
-            <li>
+            <li class="left-align">
```

Comment:
All of the other lists are center aligned on the website. I would have to update every other list on the site (which is a lot of them)